.github/workflows/npm-cd.yml

```diff
@@ -324,26 +303,44 @@ jobs:
                   npm-auth-token: ${{ secrets.NPM_AUTH_TOKEN }}
                   arch: ${{ matrix.build.ARCH }}
 
-            - name: Checkout
-              if: ${{  matrix.build.TARGET != 'aarch64-unknown-linux-musl'}}
-              uses: actions/checkout@v4
-              with:
-                  submodules: "true"
-
-            - name: Install node
-              if: ${{ !contains(matrix.build.TARGET, 'musl') }}
+            - name: Setup node
+              if: ${{ matrix.build.TARGET != 'aarch64-unknown-linux-musl' }}
               uses: actions/setup-node@v3
               with:
                   node-version: "16"
                   registry-url: "https://registry.npmjs.org"
+                  architecture: ${{ matrix.build.ARCH }}
```

Comment:
Update to node 20 (check for other places where v.16 is used)

Reply:
See comment on your previous comment